### PR TITLE
LPS-73741 adding tests

### PR DESCRIPTION
**Pull request by linolaoj**

### CODE REVIEW

**Comment by linolaoj** [comment]
ci:retest

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/forms-and-workflow/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/service/test/CalendarBookingServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+				_user1, calendar, serviceContext);
+
+		calendarBooking.setStatus(WorkflowConstants.STATUS_PENDING);
+
+		CalendarBookingLocalServiceUtil.updateCalendarBooking(calendarBooking);
+
+		int[] statuses = {
+			WorkflowConstants.STATUS_PENDING, WorkflowConstants.STATUS_DRAFT,
+			WorkflowConstants.STATUS_ANY
+		};
```
Comment (linolaoj) [inline modules/apps/forms-and-workflow/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/service/test/CalendarBookingServiceTest.java]: Hi @natocesarrego , as we talk earlier, the test can be done using only the pending status, other statuses can be removed. 
Thanks!

Comment (natocesarrego) [inline modules/apps/forms-and-workflow/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/service/test/CalendarBookingServiceTest.java]: Fine, @linolaoj. Thanks.

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/brandizzi/liferay-portal/pull/441. [See changes here.](https://github.com/natocesarrego/liferay-portal/compare/bdc9345~2...bdc9345) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/brandizzi/liferay-portal/pull/442. [See changes here.](https://github.com/natocesarrego/liferay-portal/compare/368f330~2...368f330) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**